components/inputs/input-checkbox.js

```diff
@@ -199,27 +196,33 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(RtlMix
 		};
 		const ariaChecked = this.indeterminate ? 'mixed' : undefined;
 		const disabled = this.disabled || this.skeleton;
-		const offscreenContainer = this.description ? html`<div class="d2l-offscreen" id="${this._descriptionId}">${this.description}</div>` : null;
-		const ariaDescribedByIds = `${this.description ? this._descriptionId : ''} ${this._hasInlineHelp ? this._inlineHelpId : ''}`.trim();
+		const offscreenContainer = this.description ? html`<div class="d2l-offscreen" id="${this.#descriptionId}">${this.description}</div>` : null;
+		const ariaDescribedByIds = `${this.description ? this.#descriptionId : ''} ${this._hasInlineHelp ? this.#inlineHelpId : ''}`.trim();
+		const disabledTooltip = disabled && this.disabledTooltip ?
+			html`<d2l-tooltip align="start" class="vdiff-target" for="${this.#inputId}" ?force-show="${this.isHovered}" position="top">${this.disabledTooltip}</d2l-tooltip>` :
+			nothing;
 		return html`
 			<label>
-				<span class="d2l-input-checkbox-wrapper d2l-skeletize"><input
+				<span @mouseleave="${this.#handleMouseLeave}" @mouseenter="${this.#handleMouseEnter}"><span class="d2l-input-checkbox-wrapper d2l-skeletize"><input
```

Comment:
Needed to track the user's hover state separately and force tooltip with `force-show` because we want to display it when they hover over the checkbox text, not just the input.